### PR TITLE
Add Multiple Products To Cart

### LINKED ISSUE #274: Support Adding Multiple Products To Cart
We need a JS method similar to `Nosto.addProductToCart` but for adding multiple products into cart at the same time. Magento's add to cart controller supports this. Additional product ids must be sent in `related_product` field.

```javascript
Nosto.addMultipleProductsToCart = function (productIds, element) {
    // Track cart addition 
    var fields = {
        // Form key & "main product"
        "related_product": productIds
    };
};
```

### PR DESCRIPTION
**Pull request by supercid**
Closes #274

### CODE REVIEW

**Comment by supercid** [comment]
Sample Id's for testing:
```javascript
Recobuy.addMultipleProductsToCart([
    {'productId' : '1', 'skuId' : '24-MB01'},
    {'productId' : '2', 'skuId' : '24-MB04'},
    {'productId' : '3', 'skuId' : '24-MB03'},
], this);
```
You can also query the database:
```sql
SELECT entity_id AS product_id, sku FROM catalog_product_entity
```

File: view/frontend/web/js/recobuy.js [hunk 1]
```diff
@@ -48,6 +48,18 @@ define(['catalogAddToCart', 'nostojs', 'jquery'], function (addToCart, nostojs,
         Recobuy.addSkuToCart(productData, element);
     };
 
+    // Products must be and array of objects [{productId: "123", skuId: "321"}, {...}]
+    Recobuy.addMultipleProductsToCart = function (products, element) {
+        var productData = {};
+        productData["productId"] = products.pop().productId;
+        var related = [];
+        products.forEach(function(product) {
+            related.push(product.productId);
```
[the remark this answers is not in the document]
Comment (supercid) [reply view/frontend/web/js/recobuy.js]: It does not. Magento Add To Cart Controller doesn't support Sku id's, only product id's. This is to keep consistency in how it's going to be used.

[the remark this answers is not in the document]
Comment (supercid) [reply view/frontend/web/js/recobuy.js]: Fixed, thanks 👍

File: view/frontend/web/js/recobuy.js [hunk 2]
```diff
@@ -48,18 +48,50 @@ define(['catalogAddToCart', 'nostojs', 'jquery'], function (addToCart, nostojs,
         Recobuy.addSkuToCart(productData, element);
     };
 
-    //Product object must have fields productId and skuId productId: 123, skuId: 321
+    // Products must be and array of objects [{'productId': '123', 'skuId': '321'}, {...}]
+    Recobuy.addMultipleProductsToCart = function (products, element) {
```
Comment (chenliangde) [inline view/frontend/web/js/recobuy.js]: What if the product doesn't have a sku? It should be able to add product to cart without setting skuId

**Pull request merged**